packages/flutter_driver/lib/src/extension.dart

```diff
@@ -179,23 +179,26 @@ class FlutterDriverExtension {
     return constructor(finder);
   }
 
-  Future<TapResult> tap(Tap command) async {
-    prober.tap(await _waitForElement(_createFinder(command.finder)));
+  Future<TapResult> tap(Command command) async {
+    Tap tap = command;
```

Comment:
(Specifically, this removes the ability for the analyzer to tell when the method is being called incorrectly.)